apps/cli/main.js

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env node
+
+// NOTE: for now the following needs to be in place: ./server/index.cjs, ./client, ./external
+
+const ontimeServer = require('./server/index.cjs');
+const { initAssets, startServer, startIntegrations, shutdown } = ontimeServer;
+
+async function startOntime() {
+  await initAssets();
+
+  await startServer();
+
+  await startIntegrations();
+}
+
+startOntime();
+
+process.on('SIGINT',()=>{
+  shutdown()
+})
```

Comment:
questions: 
- I think the formatting here is not following the prettier rules
- should we consider other shutdown signals? the app handles `SIGHUP` `SIGINT` and `SIGTERM`
- should we improve log to user (simple console log for now like:
```suggestion
process.on('SIGINT', () => {
  console.log('User requested Ontime shutdown');
  shutdown();
})
```